etherman/etherman.go

```diff
@@ -434,6 +441,7 @@ func (etherMan *Client) depositEvent(ctx context.Context, vLog types.Log, blocks
 	deposit.TxHash = vLog.TxHash
 	deposit.Metadata = d.Metadata
 	deposit.LeafType = d.LeafType
+	deposit.OriginRollupID = etherMan.GetRollupID()
```

Comment:
The only reliable way to know where a deposit comes from is contextual: since we are doing queries to a RPC, we know that a deposit event comes from that RPC, and we know the Rollup ID associated to that RPC